server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexerDao.java

```diff
@@ -145,7 +140,7 @@ public boolean isAllowAnyone() {
     "        AND group_id IS NOT NULL " +
     "      UNION " +
 
-    // Anyone virtual group
+    // public projects are accessible to any one
 
     "      SELECT '" + RowKind.ANYONE + "' as kind," +
```

Comment:
I'll keep ANYONE because it refers to the index field "allowAnyone" and I don't want to get into refactoring all this